infra/resources/_modules/key_vaults/io_com_key_vault.tf

```
@@ -0,0 +1,28 @@
+resource "azurerm_key_vault" "io_com_kv" {
+  name                     = "io-p-itn-com-kv"
+  location                 = var.location
+  resource_group_name      = var.resource_group_name
+  tenant_id                = var.tenant_id
+  purge_protection_enabled = false
+
+  tags = var.tags
+
+  sku_name = "standard"
+
+  access_policy {
+    tenant_id = var.tenant_id
+    object_id = var.object_id
+
+    key_permissions = [
+      "Get",
+    ]
+
+    secret_permissions = [
+      "Get",
+    ]
+
+    storage_permissions = [
+      "Get",
+    ]
+  }
```

are you sure we want to use access policies? I think we should use rbac auth (to use "roles") [like fims](https://github.com/pagopa/io-fims/blob/main/infra/resources/_modules/key_vaults/key_vault_fims.tf)

```suggestion
  enable_rbac_authorization  = true
```

---

[bc5c8c1](https://github.com/pagopa/io-messages/pull/260/commits/bc5c8c1897d93b9b7a81f242e1e802b0164bd80b)